src/index.html

```diff
@@ -13,6 +13,36 @@
   </head>
 
   <body>
+    <nav class="navbar">
+      <div class="container">
+        <div class="navbar-left">
+          <div class="navbar-brand">
+            <picture class="brand-image">
```

Comment:
I think `<picture>` is unnecessary here because we have only one version of the logo on all screens/devices.